### PR TITLE
Add example to `optuna.study.get_all_study_summaries`

### PR DESCRIPTION
**Pull request by norihitoishida**
## Motivation
This PR is related to #1334

## Description of the changes
Update the docstring of `get_all_study_summaries`

### CODE REVIEW

**Comment by norihitoishida** [comment]
I don't understand why `ci/circleci: checks` fails. 
Please give me your opinion!

**Comment by zchenry** [comment]
@norihitoishida I ran circleci check in my local environment, and found `black-20.8b1` is used. I think if you can update your `black` to the latest version (which is `20.8b1`) and run `black . --line-length 99 --exclude="docs|tutorial"` to solve this.

**Comment by norihitoishida** [comment]
Thanks @zchenry !
I love you!

**Comment by crcrpar** [comment]
> black . --line-length 99 --exclude="docs|tutorial"

Now we can omit `--line-length 99 --exclude="docs|tutorial"` if your local repository has `pyproject.toml` 😉 
Ref: https://github.com/optuna/optuna/pull/1649

**Comment by norihitoishida** [comment]
I'm wondering that we need RDBStorage or not.
Please refer to https://github.com/optuna/optuna/pull/1741#pullrequestreview-477710950.

Please give me your opinion!

File: optuna/study.py [hunk 1]
```diff
@@ -939,6 +939,32 @@ def delete_study(
...
+            import optuna
+
+            def objective(trial):
+                x = trial.suggest_float("x", -10, 10)
+                return (x - 2) ** 2
+
+            study = optuna.create_study(study_name="example-study", storage="sqlite:///example.db")
+            study.optimize(objective, n_trials=3)
+
+            study_sammary = optuna.study.get_all_study_summaries(storage="sqlite:///example.db")[0]
```
Comment (hvy) [inline optuna/study.py:962]:
```suggestion
            study_summary = optuna.study.get_all_study_summaries(storage="sqlite:///example.db")[0]
```

Comment (hvy) [inline optuna/study.py:962]: Just an idea. What do you think? We could make the `[0]` more clear and also demonstrate some types and properties involved.
```suggestion
            study_summaries = optuna.study.get_all_study_summaries(storage="sqlite:///example.db")
            assert len(study_summaries) == 1
            
            study_summary = study_summaries[0]
            assert study_summary.study_name == "example-study"
```

**Review by hvy — CHANGES_REQUESTED** [review]
Thanks for the PR, I left some comments. Would you mind taking a look?

File: optuna/study.py (hunk 1 above)
Comment (norihitoishida) [reply optuna/study.py]: Thank you for nice idea, I agree with you.

**Review by hvy — APPROVED** [review]
Oh, early morning. LGTM! Let me approve this PR although we might want to wait for https://github.com/optuna/optuna/pull/1741#pullrequestreview-477710950 as you mentioned.

**Review by HideakiImamura — APPROVED** [review]
LGTM!

**Comment by HideakiImamura** [comment]
I will wait to merge this PR until the end of the discussion in #1741.

**Comment by hvy** [comment]
I guess we can merge this PR now https://github.com/optuna/optuna/pull/1741#issuecomment-683233981 ?

**Pull request merged**